README.md

```diff
@@ -186,3 +188,123 @@ spec:
 - Check that your application data has been restored:
 
 `oc get route <route-name> -n <app-ns> -ojsonpath="{.spec.host}"`
+
+<hr style="height:1px;border:none;color:#333;">
+
+<h4> Advanced Options <a id="Advanced-Options"></a></h4>
+
+The OADP VolumeSnapshotMover feature supports volume snapshots via the CSI driver. 
+Both CephRBD and CephFS are [supported via CSI](https://github.com/ceph/ceph-csi).
+The OADP VolumeSnapshotMover feature leverages some of the more recently added 
+features of Ceph and CSI to be performant in large scale environments.
+
+One of these newly added features for backups with CephFS to be more performant
+is the [shallow copy](https://github.com/ceph/ceph-csi/blob/devel/docs/design/proposals/cephfs-snapshot-shallow-ro-vol.md) 
+method, which is available > OCP 4.12.
+
+In large scale backups the OADP highly recommends OCP 4.12 and above. In OCP 4.12
+extra parameters on the DPA are required for this CephFS shallow copy.
+These parameters should not be required in OCP >= 4.13. 
+
+For backups on CSI backed by CephFS using shallow copy, OADP requires the 
+following volume options specified in the DPA.
+
+```
+volumeOptions:
+  sourceVolumeOptions:
+    accessMode: ReadOnlyMany
+    cacheAccessMode: ReadWriteMany
+    cacheStorageClassName: ocs-storagecluster-cephfs
+    moverSecurityContext: true
+    storageClassName: ocs-storagecluster-cephfs-shallow
+```
+
+Since the DPA is a cluster wide configuration, if you plan to backup any other
```

Comment:
Not a bad plan but I am still unsure about having multiple DPA's for this use case, multiple DPA's means multiple instances of velero and vsm controller with the same backup repository (most likely) and single instance of volsync controller. Maybe the modify the DPA per use case ? I am still unsure. The user can specify the storageclass they want use via volumeOptions, right ? Shouldn't we suggest that as an alternative ?

Reply:
well... Rock vs. Hard Place imho.  Asking customers to update their DPA per backup at backup time is not OK w/ me.  Since Velero can only process one backup at a time, I would *think* having two dpa's might be the better solution.  I don't know, I'm new here.

Reply:
I agree with you. Now looking to solve this use case via the the volumeoptions field I think we should somehow tie the datamover volumeoptions with the backup level and not at the upper install level (OADP Oeprator/DPA/Velero level)

Reply:
@weshayutin I'm a bit out of the loop for some of this, but two DPAs is only appropriate if the intent is two velero instances in two different namespaces. If that's not what you're proposing here, then we probably need to come up with something different.

Reply:
@shubham-pampattiwar ++ Having the volumeoptions tied to the backup cr vs. the dpa is HUGE "YES PLEASE"